client/my-sites/site-settings/style.scss

```diff
@@ -586,6 +586,16 @@ button.site-settings__general-settings-set-guessed-timezone.button.is-borderless
 	pointer-events: none;
 }
 
+#site-privacy-settings .inline-support-link {
+	color: var(--color-text-subtle);
+
+	svg {
+		vertical-align: middle;
+		height: 18px;
```

Comment:
If we are passing the same height and width, we can use the `iconSize` prop that is already present in the `InlineSupportLink` instead of adding height and width. This will set both height and width to `18px`:

```
				<InlineSupportLink
									supportContext="privacy"
									showText={ false }
									iconStyle="info-outline"
									iconSize={ 18 }
								/>
```